app/models/wpcc/calculator_delegator.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_model'
```

Comment:
is this require needed? I thought this was something that was loaded in the rails pipeline